src/plugins/intel_npu/src/compiler/src/zero_compiler_in_driver.cpp

```diff
@@ -374,19 +371,23 @@ std::string LevelZeroCompilerInDriver<TableExtension>::serializeIOInfo(const std
     inputsPrecisionSS << INPUTS_PRECISIONS_KEY << KEY_VALUE_SEPARATOR << VALUE_DELIMITER;
     inputsLayoutSS << INPUTS_LAYOUTS_KEY << KEY_VALUE_SEPARATOR << VALUE_DELIMITER;
 
-    for (const std::shared_ptr<ov::op::v0::Parameter>& parameter : parameters) {
-        const std::string& name = parameter->get_friendly_name();
-        const ov::element::Type& precision = parameter->get_element_type();
-        const size_t rank = parameter->get_shape().size();
+    if (!parameters.empty()) {
```

Comment:
Is there a test where this case can be caught?

Reply:
We're adding a private test. I will tag you in the PR.